pages/04.knowledgebase/01.installation/how-to-install-mautic-on-a-localhost/how-to-install-mautic-using-ddev/item.md

```diff
@@ -106,26 +163,11 @@ and
 respectively to turn Xdebug on and off.
 
 ### Changing PHP versions
-Navigate to `.ddev/config.yaml` and edit the parameter called `php_version`. Once that is saved, run 
+Navigate to `.ddev/config.yaml` and edit the parameter called `php_version`. **(Be sure to also change the version number on any additional packages you have set on the line `webimage_extra_packages`)**. Once that is saved, run 
 
 `ddev restart`
 
-### Using additional PHP modules
```

Comment:
Was there a reason for removing this section on additional PHP modules?  Not sure if the omission was intentional!

Reply:
It was intentional :) Since adding extra modules was already covered previously in the guide (when the `php73-imap` was added), I felt it was confusing that it was suddenly re-mentioned down here, so instead I added a remark up there mentioning how the user can add any additional PHP modules they want.